common/e2e.go

```diff
@@ -60,8 +61,7 @@ func RunE2ETests(t *testing.T, cfg *config.Config) {
 	}
 
 	// setup reporter
-	err = os.Mkdir(cfg.ReportDir, os.ModePerm)
-	if err != nil {
+	if err = os.Mkdir(cfg.ReportDir, os.ModePerm); err != nil {
```

Comment:
I like the readability of this more and I've been changing those as I run into them. /shrug